src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +41,37 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${id}`}
             </div>
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button
+            <i
```

Comment:
please fix previous comment about test. Why didn't you fix it? Maybe you forgot to push your changes? If you have any problems fill free to ask in fe_chat. 
 Tag `i` is only used to style text. You should not use it as a button or hang handlers on it
 Please fix all cases in your solution